src/elements-experimental/timetable-row/timetable-row.visual.spec.ts

```diff
@@ -69,6 +71,7 @@ describe(`sbb-timetable-row`, () => {
     { name: 'bus', trip: busTrip },
     { name: 'ship', trip: shipTrip },
     { name: 'walkTime', trip: walkTimeTrip },
+    { name: 'a11yFootpath', trip: a11yFootpathTrip },
```

Comment:
Otherwise not visible at https://lyne-components-visual-regression-diff-pr3048.app.sbb.ch/compare/sbb-timetable-row/a11yFootpath
```suggestion
    { name: 'a11yFootpath', trip: a11yFootpathTrip, a11yFootpath: true },
```